ontology/yaml/resources/HVAC/entity_types/FRZ.yaml

```diff
@@ -0,0 +1,56 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the License);
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an AS IS BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+########################
+### Canonical Types  ###
+########################
+
+### FREEZER ###
+
+FRZ_SS:
```

Comment:
Not sure why you have broken out each freezer with only a single abstract type. Please combine into the actual canonical type you plan to use for the freezers you have.

Reply:
The individual canonicals have been removed and created a single one.